apps/sms/views/conversation/js/conversation.js

```diff
@@ -129,15 +133,27 @@ var ConversationView = {
                               this.onVisibilityChange);
 
     this.toField.addEventListener(
-      'keypress', this.toFieldKeypress.bind(this), true
+      'keypress', 
+      Utils.throttle(this.toFieldKeypress.bind(this),
```

Comment:
and you can set the option as parameter if you apply https://github.com/mozilla-b2g/gaia/pull/31677/files#r38887892